Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Themes/InteractableThemePropertySettings.cs

```diff
@@ -47,6 +47,8 @@ public struct InteractableThemePropertySettings
         public InteractableThemeBase Theme;
         public List<InteractableThemeProperty> Properties;
         public List<InteractableThemeProperty> History;
+        public List<InteractableCustomSetting> CustomSettings;
```

Comment:
What are these for? They seem unrelated to the PR description.

Reply:
It was included in the list, but could have been separate. This was something that was stubbed in a few places but never finished and I use them to do the InteractableGrabScaleTheme, plus a couple other themes that have been removed from this PR.